lib/cadet_web/router.ex

```diff
@@ -169,6 +169,19 @@ defmodule CadetWeb.Router do
     )
   end
 
+  # Notifications endpoints
```

Comment:
How are these APIs being called from the frontend? Why are they not piped through authentication and course-specific assignment? I am not sure what might happen here. If some request is fired from a student that already has his log in session expired or if the student changed course(which might lead to some cache issue or privilege issue)...

Reply:
Also please separate the admin routes from the student routes